helm-chart/dicom-adapter/Chart.yaml

```diff
@@ -17,7 +17,7 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for the DICOM Adapter
 name: dicom-adapter
-version: 0.0.0.0
+version: 0.8.1
```

Comment:
Do we want to replace it with a script instead of changing this for every build/release?

Reply:
It would be ideal to separate repo, then we could do the following:
`helm repo add dicomadapter https://github.com/nvidia/clara-dicom-helm-chart` 
`helm show values`
`helm install -f ./values.yaml dicomadapter`